code/modules/mob/living/silicon/robot/drone/maint_drone.dm

```diff
@@ -113,6 +122,16 @@
 
```

Comment:
Might want to think about whether we still want drones to be able to take ghost role spawns, especially ones that used to be a pAI.